packages/go/util/analyze.go

```diff
@@ -223,7 +225,7 @@ func main() {
 		if fn.Name.IsExported() == true {
 			for _, param := range fn.Type.Params.List {
 				paramStr := fmt.Sprintf("%s", param.Type)
-				if strings.Contains(string(paramStr), "http ResponseWriter") && len(fn.Type.Params.List) == 2 {
+				if strings.Contains(string(paramStr), "http ResponseWriter") && len(fn.Type.Params.List) == 2 && (fn.Recv == nil || len(fn.Recv.List) == 0) {
```

Comment:
Should this fallback logic also check for `strings.Contains(string(params), "*http.Request")`, or is that absent because of the parsing issue this fallback exists to solve?

Reply:
@jarneson do you have any idea why this condition was not included here? I can't tell if it's intentional or accidental.